docs/_docs/monitoring-metrics/new-metrics.adoc

```diff
@@ -286,23 +286,28 @@ Register name: `io.communication`
 |SentBytesCount | long  |  Sent bytes count.
 |ReceivedBytesCount|  long|    Received bytes count.
 |ReceivedMessagesCount|   integer| Received messages count.
-|RejectedSslSessionsCount|   integer|   TCP sessions count that were rejected due to the SSL errors (metric is exported only if SSL is enabled).
+|RejectedSslSessions|   integer|   TCP sessions count that were rejected due to the SSL errors (metric is exported only if SSL is enabled).
 |SslEnabled|   boolean|   Indicates whether SSL is enabled.
 |SslHandshakeDurationHistogram|   histogram|   Histogram of SSL handshake duration in milliseconds (metric is exported only if SSL is enabled).
 |===
 
 
 == Ignite Thin Client Connector
 
-Register name: `connector.client.thin.tcp`
+Register name: `client.connector`
 
 
 [cols="2,1,3",opts="header"]
 |===
 |Name|    Type|    Description
 |ActiveSessionsCount|   integer|   Active TCP sessions count.
 |ReceivedBytesCount|   long|   Received bytes count.
-|RejectedSslSessionsCount|   integer|   TCP sessions count that were rejected due to the SSL errors (metric is exported only if SSL is enabled).
+|RejectedSslSessions|   integer|   TCP sessions count that were rejected due to the SSL errors (metric is exported only if SSL is enabled).
+|RejectedSessionsByTimeout|   integer|   TCP sessions count that were rejected due to handshake timeout.
+|RejectedSessionsDueAuthentication|   integer|   TCP sessions count that were rejected due to failed authentication.
+|RejectedSessionsTotal|   integer|   Total number of rejected TCP connections.
+|{clientType}.AcceptedSessions|   integer|   Number of successfully established sessions.
```

Comment:
{clientType}.AcceptedSessions
{clientType}.ActiveSessions
Let's point in the description that it's number of sessions that belongs to particular connection type.

Reply:
Fixed